util/regexp-assemble/data/942180.data

```diff
@@ -44,3 +47,22 @@
 \sis\s*?0\W
 where\s[\s\w\.,-]+\s=
 [\"'`][<>~]+[\"'`]
+
+##! Added (?:\/\*.*?\*\/)?\s* to support comment block that can be used to bypass the filter
+##! Replaced lazy \s? to greedy \s*
+
+\d[\"'`]\s+(?:\/\*.*?\*\/)?\s*[\"'`]\s+(?:\/\*.*?\*\/)?\s*\d
+^admin\s*(?:\/\*.*?\*\/)?\s*[\"'`]
+
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*or[\w\s-]+\s*(?:\/\*.*?\*\/)?\s*[+<>=(),-]\s*(?:\/\*.*?\*\/)?\s*[\d\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*xor[\w\s-]+\s*(?:\/\*.*?\*\/)?\s*[+<>=(),-]\s*(?:\/\*.*?\*\/)?\s*[\d\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*div[\w\s-]+\s*(?:\/\*.*?\*\/)?\s*[+<>=(),-]\s*(?:\/\*.*?\*\/)?\s*[\d\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*like[\w\s-]+\s*(?:\/\*.*?\*\/)?\s*[+<>=(),-]\s*(?:\/\*.*?\*\/)?\s*[\d\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*between[\w\s-]+\s*(?:\/\*.*?\*\/)?\s*[+<>=(),-]\s*(?:\/\*.*?\*\/)?\s*[\d\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*and[\w\s-]+\s*(?:\/\*.*?\*\/)?\s*[+<>=(),-]\s*(?:\/\*.*?\*\/)?\s*[\d\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*[^\w\s]?=\s*(?:\/\*.*?\*\/)?\s*[\"'`]
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*[!=|][\d\s!=+-]+.*?[\"'`(].*?$
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*[!=|][\d\s!=]+.*?\d+$
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*like\W+[\w\"'`(]
+\sis\s*(?:\/\*.*?\*\/)?\s*0\W
+where\s+(?:\/\*.*?\*\/)?\s*[\s\w\.,-]+\s+(?:\/\*.*?\*\/)?\s*=
```

Comment:
Same as in 942120.data: the required space must be either before *or* after the comment.

Reply:
Agreed.